src/core/translation/locales/en/navigation.json

```diff
@@ -0,0 +1,38 @@
+{
+        "nav_dashboard": "Dashboard",
```

Comment:
please turn on you eslint and format your file with Eslint/prettier
(2 spaces instead of 4 spaces)